src/ralph/ssl_certificates/tests/samples/allegro.pl.vte1.te4.local.crt

```diff
@@ -0,0 +1,115 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
isn't it better to generate new, dedicated certs for testing purpose?

Reply:
You have right, I'll generate it